src/components/controls/controls.tsx

```diff
@@ -94,7 +135,6 @@ function Controls({ treeOn, mapOn, frequenciesOn, measurementsOn, mobileDisplay
       <span style={{ paddingTop: "10px" }} />
       <AnnotatedHeader title={t("sidebar:Panel Options")} tooltip={PanelOptionsInfo} mobile={mobileDisplay}/>
```

Comment:
Good catch, I've addressed it here: 9714d4234bac0ea17c81ee1e00957b1583ab9c4a

> An alternate approach would be to combine the Animation Options + Panel Options into a "Other options" section.

My inclination is to avoid binning of irrelevant options together in an "other" category. Since the options would be harder to find, it seems like this idea only makes sense under the assumption that these options are used by few users - something we don't have the user data to confirm.

I have an idea to toggle the Animation Options, I'll try that out in another PR.

Reply:
> I have an idea to toggle the Animation Options, I'll try that out in another PR.

#1705